### DISCUSSION TITLE
2.3.9 Released

### DISCUSSION THREAD

**Discussion by jarena3** [massif-press/compcon#2051] [Archive]
### Bug Fixes
- **Mech Sheet** Mechs should now correctly save changed equipment on reload/when logged out. Thanks to @msprijatelj for the help! [#1989](https://github.com/massif-press/compcon/issues/1989)
- **Active Mode** Correct a bug that was preventing all actions from being populated and not respecting their correct use state [#2005](https://github.com/massif-press/compcon/issues/2005) [#2037](https://github.com/massif-press/compcon/issues/2037)
- **Help Menu** Update help menu options [#2001](https://github.com/massif-press/compcon/issues/2001)
- **Active Mode** Improve Overcharge Pip determination (thanks @msprijatelj!)
- **Cloud** Improve sign-up/in reliability (thanks @msprijatelj!)
- **Pilot Sheet** Fixed failing import for Pilots with Bond data being imported into environments without the KTB LCP installed [#2007](https://github.com/massif-press/compcon/issues/2007)

### Features
- **Global** Improved Keyboard accesibility ((thanks @Qazzquimby! First PR! 🎉) [#1557](https://github.com/massif-press/compcon/issues/1557)
- **Pilot Sheet** Revised Statblock Generator (thanks @karidyas!) [#1887](https://github.com/massif-press/compcon/issues/1887)
- **LCP Importer** Humanize Reserves/Factions (thanks @msprijatelj!) [#1557](https://github.com/massif-press/compcon/issues/1557)
- **Nav** Add Help Icon, FAQ links, style fixes (thanks @msprijatelj!) [#2033](https://github.com/massif-press/compcon/issues/2033)

**hotfix `a` deployed; this should address an out-of-memory issue**

**Comment by IkomaTanomori** [comment]
Downloading this version caused the app to grind to a halt and give an out of memory error very quickly every load. OS: windows 11. Browser: chrome, with app downloaded, not running in a tab. LCPs used: official + guide to liminal space.

**Reply by jarena3** [reply]
I'm deploying a hotfix 2.3.9a now that should hopefully take care of this. Please let me know if it still keeps happening.

**Reply by IkomaTanomori** [reply]
Downloaded the hotfix, it appears to have fixed the issue.
